stream-transformation-tool-service/src/test/java/uk/gov/justice/tools/eventsourcing/transformation/service/EventStreamTransformationServiceTest.java

```diff
@@ -252,7 +252,7 @@ public void shouldPerformAllTheIndicatedActionsOnAStream() throws EventStreamExc
     }
 
     private JsonEnvelope buildEnvelope(final String eventName) {
-        return DefaultJsonEnvelopeProvider.provider().envelopeFrom(
+        return new DefaultJsonEnvelopeProvider().envelopeFrom(
```

Comment:
The best way to create an envelope is like this:

import static uk.gov.justice.services.messaging.JsonEnvelope.envelopeFrom;
import static uk.gov.justice.services.messaging.JsonEnvelope.metadataBuilder;
...

    final UUID id = randomUUID();
    final String commandName = "notification-added";

    final JsonEnvelope envelope = envelopeFrom(
            metadataBuilder()
                    .withId(id)
                    .withName(commandName)
    createObjectBuilder()
            .add("exampleField", "example value"));